packages/jest-console/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Unreleased (2.0.0)
```

Comment:
It will also make `wp-scripts test-unit-js` fail when code uses `console.log` and `console.info`, so we will have to bump `@wordpress/scripts` - ideally we should align it with Babel 7 upgrade.